app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.kt

```diff
@@ -727,6 +731,8 @@ class UrlInputFragment :
     }
 
     override fun onSharedPreferenceChanged(sharedPreferences: SharedPreferences, key: String) {
-        if (key == activity?.getString(R.string.pref_key_homescreen_tips)) { updateTipsLabel() }
+        if (key == activity?.getString(R.string.pref_key_homescreen_tips)) {
```

Comment:
It would be preferable that the changes not related to this PR, especially indentation or code rearrangements are not included here.